frontend/package.json

```diff
@@ -10,8 +10,11 @@
     "@types/node": "^16.11.36",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.4",
+    "axios": "^0.27.2",
+    "interweave": "^13.0.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
Die drei neuen Dependencies werden noch nicht verwendet. Üblicherweise würde ich vorschlagen sie zu entfernen - aber in diesem Fall gehe ich davon aus, dass Du die Dependencies in Kürze brauchst. Stimmt's?

Reply:
Richte, hier handelt es sich um die Überbleibsel aus meinem frontend-Prototype. Ich habe die eigentlichen Frontendcomponenten nur für den Pullrequest herausgenommen, um den Pullrequest nicht zu groß zu machen.